zproject/default_settings.py

```diff
@@ -266,6 +267,9 @@
 # purpose now that the REALMS_HAVE_SUBDOMAINS migration is finished.
 SYSTEM_ONLY_REALMS = {"zulip"}
 
+# Default deadline for demo organizations
+DEMO_ORG_DEADLINE_TIMEDELTA = datetime.timedelta(days=30)
```

Comment:
Our standard approach in naming settings is to do e.g. `DEMO_ORG_DEADLINE_DAYS = 30`; that's why this file didn't import `datetime` previously.

Reply:
@timabbott Done! :)